dirsrvtests/tests/suites/webui/__init__.py

```diff
@@ -120,3 +120,107 @@ def fin():
         remove_instance_through_webui(topology_st, page, browser_name)
 
     request.addfinalizer(fin)
+
+
+def create_entry(frame, entry_type, entry_data):
+    if entry_type == 'user':
+        frame.get_by_role("tabpanel", name="Tree View").get_by_role("button", name="Actions").click()
+        frame.get_by_role("menuitem", name="New ...").click()
+        frame.get_by_role("button", name="Next").click()
+        frame.get_by_role("button", name="Options menu").click()
+        frame.get_by_role("option", name="Posix Account").click()
+        frame.get_by_role("button", name="Next", exact=True).click()
+        frame.get_by_role("button", name="Next", exact=True).click()
+        frame.get_by_role("button", name="Place row 0 in edit mode").click()
+        frame.get_by_role("textbox", name="_").fill(entry_data['user_name'])
+        frame.get_by_role("button", name="Save row edits for row 0").click()
+        frame.get_by_role("button", name="Place row 1 in edit mode").click()
+        frame.get_by_role("textbox", name="_").fill(entry_data['user_name'])
+        frame.get_by_role("button", name="Save row edits for row 1").click()
+        frame.get_by_role("button", name="Place row 2 in edit mode").click()
+        frame.get_by_role("textbox", name="_").fill(entry_data['user_id'])
+        frame.get_by_role("button", name="Save row edits for row 2").click()
+        frame.get_by_role("button", name="Place row 3 in edit mode").click()
+        frame.get_by_role("textbox", name="_").fill(entry_data['user_home_dir'])
+        frame.get_by_role("button", name="Save row edits for row 3").click()
+        frame.get_by_role("button", name="Place row 4 in edit mode").click()
+        frame.get_by_role("textbox", name="_").fill(entry_data['user_id'])
+        frame.get_by_role("button", name="Save row edits for row 4").click()
+        frame.get_by_role("button", name="Place row 5 in edit mode").click()
+        frame.get_by_role("textbox", name="_").fill(entry_data['user_id'])
+        frame.get_by_role("button", name="Save row edits for row 5").click()
```

Comment:
I think a lot of this code can be packed with a 'for loop'.

Reply:
I don't see any reason why this test is failing. I didn't modify it in any way and it was passing before. And even now when i try to run this test in a VM, it is passing without any issue. I will look into it tomorrow when the tests for new commit are finished.

Also I forgot to edit the test steps in docstring to reflect the new "load_ldap_browser_tab" function.